.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,40 @@
+name: Bug report
+description: Report a bug. For security vulnerabilities see Report a security vulnerability in the templates.
+title: "BUG:"
+labels: [00 - Bug]
+
+body:
+- type: textarea
+  attributes: 
+    label: "Describe the issue:"
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: "Reproduce the code example:"
+    description: |
+      A short code example that reproduces the problem/missing feature. It should be self-contained, i.e., possible to run as-is via
+      `python myproblem.py`.
+    placeholder: |
+      python
+      import numpy as np
+      << your code here >>
```

Comment:
This is missing the newlines in the placeholder.  I think we can skip the `python`.

Reply:
> This is missing the newlines in the placeholder.

I am not sure I get what you mean. Can you please elaborate?

Reply:
All of these end up on a single line without spaces when I go to your fork.  Or is that an older version?

Reply:
It could be the older version that you saw because right now this is what I see:
![image](https://user-images.githubusercontent.com/60316606/132692668-c89f81f5-ba62-429b-9533-6913b052fb20.png)

Reply:
OK, still getting it on one line, but I am willing to blame it on buggy github + browser variations if it looks right to for you, thanks.  I think we still should remove the `python` from the first line.

(Additional nit, we can delay)  I now see that we do write `python myproblem.py`, I think we should maybe write: can be copy-pasted into the python interpreter or run as.  Just noting this because I remember 2-3 occasions were I was wondering why users bothered to provide actual scripts for 10 line snippets, and this might be it ;).